### DISCUSSION TITLE
inventory menu / deluxe menu / command panels / TNML GUI

### DISCUSSION THREAD

**Discussion by thesquidghost** [QuickShop-Community/QuickShop-Hikari#1696] [Ideas]
Hello! So glad that creatorfromhell took over this. 

It would be nice to have some configuration settings wherein users could disable the chat control panel and opt-in to using a more modern inventory-style menu. At the minimum, right clicking a shop sign should just open up a inventory menu wherein changing price, buy or sell mode, managing staff, and deleting the shop is available.

**Comment by TomLewis** [comment]
+1 for using a GUI instead of the chat control panel.

The using chat to write in an amount is not user friendly when they interact with signs.

**Comment by creatorfromhell** [comment]
Completed in the latest dev build.
